DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -28,7 +27,9 @@ public IActionResult Index()
             var userDelegateId = User.GetCustomClaim(CustomClaimTypes.LearnCandidateId);
             var (adminUser, delegateUser) = userService.GetUsersById(userAdminId, userDelegateId);
 
-            var model = new MyAccountViewModel(adminUser, delegateUser);
+            var customPrompts = customPromptsService.GetCustomPromptsForCentreByCentreId(delegateUser?.CentreId ?? adminUser?.CentreId);

```

Comment:
It looks to me like the CustomPrompts will always be obtained as long as the user has either an admin or a delegate account. Is that the case? If so, is there a particular need to grab them when no delegate account is found, given we are only displaying them when there is a delegate account?

Reply:
Yeah I think I have the logic here slightly wrong. Should just be trying to pass in delegateUser?.CentreId